app/utils/url.js

```diff
@@ -125,7 +125,7 @@ export function buildUrl(urlParams = {}) {
 }
 
 export function extractEncodedTitle(url) {
-	return url ? url.replace(/^(http:\/\/[^\/]+)?(\/wiki)?\//, '') : '';
```

Comment:
Are you sure about this? How is it related to the update?

Reply:
updated eslint rules, no-useless-escape